README.md

```diff
@@ -63,7 +63,7 @@ the `Install-Script` command.
 
 **Alternatively** you can download and execute the script directly from the web
 without the use of `Install-Script`.  However we **highly recommend** that you
-[read the script](https://github.com/PowerShell/vscode-powershell/blob/develop/scripts/Install-VSCode.ps1)
+[read the script](https://git.io/vbxjj)
```

Comment:
I appreciate the shortlink but since it's not named (just some random chars), users may be hesitant to click. Can you use the normal url:

```
https://raw.githubusercontent.com/PowerShell/vscode-powershell/master/scripts/Install-VSCode.ps1
```

Reply:
The shortlink is the same as used below in line 70. I think that keeping this unconsistent is wrong. (On the other side someone responsible for Powershell Gallery Install-VSCode should update script to match the version from master) Don't you think I should change the code in line 70 to use full url as well?

Reply:
Good point - I didn't see that line. Lets change it on line 70 as well.

I'll publish a new version of the script sometime this week.